TextileNode.ts

```diff
@@ -66,6 +66,14 @@ export default {
     return threads
   },
 
+  addExternalThreadInvite: async function (name: string, pubKey: string): Promise<string> {
```

Comment:
Just learning here for when I integrate this into the app... What is the string being returned?

Reply:
it will be most of the link you'll need. so it should look like,

`https://www.textile.photos/invites/new#invite=QmTdt36G6RXktJ9UypUftiqGdogidNBmxwwNcHW39Jg3WB&key=17TxeCbotq7wzm9AsRpcRIH817617TxeCCn1iOQjQCyz&name=TestTest`

i think @sanderpick was going to change the `invite` key, but it shouldn't matter since the RN will never need to get into that. I'd also like RN to add a final parameter to the url, `&from=andrewxhill` with some identifier of the invitee so that we can use it to give context in the message of the invited